dces-drc-integration/src/main/resources/application.yaml

```diff
@@ -56,10 +56,12 @@ scheduling:
   # default is disabled. "-"
   # Example to schedule daily file processing at 8pm Mon to Sat
   # "0 0 20 * * MON-SAT"
-  fdcDailyFiles:
-    cron: ${DCES_CRON_FDC}
-  contributionsDailyFiles:
-    cron: ${DCES_CRON_CONTRIBUTIONS}
+  cron:
+    process-fdc-files: ${DCES_CRON_FDC:-}
+    process-contributions-files: ${DCES_CRON_CONTRIBUTIONS:-}
+  lock:
+    at-least: 30s
+    at-most: 15m
```

Comment:
But as the jobs run once per day, I saw no harm in making this quite long. The 'lockAtMostFor' is really for if the application crashes. If the application runs normally it will release the lock itself anyway.